lib/index.js

```diff
@@ -31,6 +32,8 @@ Filterable.prototype.query = function(s, options, callback) {
     .nodeify(callback);
 };
 
+Filterable.prototype.querySync = deasync(Filterable.prototype.query);
```

Comment:
I really don't like the idea of doing this, `deasync` changes the behaviour of the event loop, we should juste keep this async and adapt tests to test the async method.